modules/project-docs/examples/Migrating.java

```diff
@@ -66,7 +66,7 @@ public static void main(String... args) {
       // tag::timeoutbuilder[]
       // SDK 3 equivalent
       ClusterEnvironment env = ClusterEnvironment.builder()
-          .timeoutConfig(TimeoutConfig.kvTimeout(Duration.ofSeconds(5))).build();
+          .build();
```

Comment:
Is this correct?
Looks like this is now creating an empty cluster environment.

If this doesn't work, probably best to remove the whole section entirely.

That said https://docs.couchbase.com/java-sdk/current/ref/client-settings.html#config-builders does also suggest it exists?

Reply:
Someone with more knowledge than me would have to comment... User that logged the ticket seemed to think it didn't exist so I thought it would be an easy fix! 

> There is no .timeoutConfig method behind the .builder() method...

Reply:
Hmm, don't see it in the API docs https://docs.couchbase.com/sdk-api/couchbase-java-client/com/couchbase/client/java/env/ClusterEnvironment.Builder.html

So perhaps this needs checking with SDK engineers and updating throughout docs?

Reply:
Do you know who we could ask as far as developers go to verify?